schemas/tesla_app_schema.js

```diff
@@ -0,0 +1,53 @@
+// MongoDB Schema
+
+user = {
+  "_id": ObjectId("5ddec02bc23c74117af22dac"),
+  "username": "devintyler26",
+  "email": "dtyler.cunningham@gmail.com",
+}
+
+vehicle = {
+  "_id": ObjectId("5678"),
+  "id": 55619287574696016,
+  "vehicle_id": 1154919923,
+  "vin": "5YJ3E1EB5KF386157",
+  "display_name": "Tyl3r",
+  "option_codes": "AD15,...",
+  "color": None,
+  "tokens": [
+    "3f97205e77a67ca7",
+    "08e2f93583d415dc"
+  ],
+  "state": "asleep",
+  "in_service": False,
+  "id_s": "55619287574696016",
+  "calendar_enabled": true,
+  "api_version": 7,
+  "backseat_token": None,
+  "backseat_token_updated_at": None,
+  "user_ids": [ObjectId("5ddec02bc23c74117af22dac"),]
```

Comment:
A vehicle can belong to multiple users?

Reply:
Yes, e.g. It could belong to me or Kylie

Reply:
Is your plan to allow others to sign up?  What happens if the car is sold?

Reply:
But then they would see the history of the car, no?  You probably don't want that.

Reply:
That's true. So if a vehicle changes ownership, create a new vehicle document?

Reply:
I think the vehicle ID from Tesla actually changes when this happens so you could potentially make that part of your primary key.  I forget if you can have composite PKs in Mongo.